src/pages/Attendance.jsx

```diff
@@ -1,11 +1,98 @@
+import React from "react";
+import dayjs from "dayjs";
+import { Stack, styled } from "@mui/material";
+import { LocalizationProvider } from "@mui/x-date-pickers/LocalizationProvider";
+import { AdapterDayjs } from "@mui/x-date-pickers/AdapterDayjs";
+import { MobileDatePicker } from "@mui/x-date-pickers/MobileDatePicker";
+import CalendarMonthIcon from "@mui/icons-material/CalendarMonth";
+import ChurchIcon from "@mui/icons-material/Church";
+import ArrowLeftIcon from "@mui/icons-material/ArrowLeft";
+import ArrowRightIcon from "@mui/icons-material/ArrowRight";
+import SettingsIcon from "@mui/icons-material/Settings";
 import Layout from "../components/Layout/Layout";
+import Title from "../components/shared/Title";
 
 const Attendance = () => {
+  const [value, setValue] = React.useState(new Date());
+  const [openCalendar, setOpenCalendar] = React.useState(false);
+
+  const handleChange = (newData) => {
+    setValue(newData);
+    setOpenCalendar(false);
+  };
+
+  const settings = {
+    MuiPickersToolbar: {
+      styleOverrides: {
+        root: {
+          backgroundColor: "#000",
+        },
+      },
+    },
+  };
+  const handleCalendarClick = () => {
+    setOpenCalendar(true);
+  };
   return (
     <Layout>
-      <div>Attendance</div>
+      <AttendanceWapper>
+        <TitleWapper sx={{ height: "20%" }}>
+          <ChurchIcon sx={{ fontSize: 40 }} />
+          <Title>출석</Title>
+        </TitleWapper>
+        <CalendarWapper sx={{ height: "10%" }}>
+          <ArrowLeftIcon />
+          <LocalizationProvider dateAdapter={AdapterDayjs} {...settings}>
```

Comment:
```suggestion
          <LocalizationProvider dateAdapter={AdapterDayjs} {...settings} adapterLocale="ko">
```
여기 settings가 유효하게 잘 들어가고 있어?

달력 쪽 한글 포맷으로 바꾸는건 adpaterLocale 추가하고 

import "dayjs/locale/ko"; 이거 import 하면 됨.
검색 서칭 더 많이많이 해보도록